### PR TITLE
Remove plugin duplicates & lower case IoTDB in artifactId of POM

### PR DESCRIPTION
**Pull request by wujysh**
- Fix: remove plugin items which are duplicated in the same pom
- Fix: artifactId should be lowercase letters

### CODE REVIEW

File: iotdb/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	    <relativePath>../pom.xml</relativePath>
     </parent>
 
-	<artifactId>IoTDB</artifactId>
+	<artifactId>iotdb</artifactId>
```
Comment (MyXOF) [inline iotdb/pom.xml]: Should we change to lower case? @jixuan1989

**Comment by MyXOF** [comment]
I have changed some codes, just to retrigger the pipeline

File: iotdb/pom.xml (hunk 1 above)
Comment (wujysh) [reply iotdb/pom.xml]: Other sub-packages, iotdb-cli, iotdb-grafana, are all lowercased.

Comment (jixuan1989) [reply iotdb/pom.xml]: maybe lower case is better. I check Apache Hadoop, and all modules are low case...

**Pull request merged**